com_migratetojoomla/admin/src/View/Main/HtmlView.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Joomla\Component\MigrateToJoomla\Administrator\View\Main;
+
+defined('_JEXEC') or die;
+
+use Joomla\CMS\Language\Text;
+use Joomla\CMS\Factory;
+use Joomla\CMS\MVC\View\HtmlView as BaseHtmlView;
+use Joomla\CMS\Toolbar\ToolbarHelper;
+/**
+ * @package     Joomla.Administrator
+ * @subpackage  com_migratetojoomla
+ *
+ * @copyright     (C) 2023 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license       GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+/**
+ * Main "Migrate To Joomla" Admin View
+ */
+class HtmlView extends BaseHtmlView
```

Comment:
If you use forms, use the ```FormView``` as parent class.

Reply:
done